.rubocop.yml

```diff
@@ -0,0 +1,40 @@
+Lint/DuplicateMethods:
```

Comment:
I would say we can go with the _default_ community options, so no need for this file.